README.md

```diff
@@ -57,27 +57,39 @@ Now run webpack. Either everything is green and the build continues or `joi` wil
 #### CLI
 For CLI usage you probably want to install the tool globally (`npm install -g webpack-validator`) first. Then just run `webpack-validator <your-config>`.
 
-#### Customizing
+### Customizing
+#### Schema
 If you need to extend the schema, for example for custom top level properties or properties added by third party plugins like `eslint-loader` (which adds a toplevel `eslint` property), do it like this:
 
 ```js
 const validate = require('webpack-validator')
-const schema = require('webpack-validator').schema
+const Joi = require('webpack-validator').Joi
 
-// joi is installed as dependency of this package and will be available in node_modules
-// if you use npm 3. Otherwise install it explicitly.
-const Joi = require('joi')
-
-const yourSchema = schema.concat(Joi.object({
+// This joi schema will be `Joi.concat`-ed with the internal schema
+const yourSchemaExtension = Joi.object({
   // this would just allow the property and doesn't perform any additional validation
   eslint: Joi.any()
-}))
+})
 
 const config = { /* ... your webpack config */ }
 
-// Override default config by supplying your config as second parameter.
-module.exports = validate(config, yourSchema)
+module.exports = validate(config, { schemaExtension: yourSchemaExtension })
```

Comment:
Can you think of a case where it would be reasonable to override some existing rule? If not, this can work.